lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -46,14 +46,22 @@ def call(env)
           # configure the Rack middleware once
           configure()
 
-          # start a new request span and attach it to the current Rack environment;
-          # we must ensure that the span `resource` is set later
-          request_span = @tracer.trace(
-            'rack.request',
+          trace_options = {
             service: @service,
             resource: nil,
             span_type: Datadog::Ext::HTTP::TYPE
-          )
+          }
+
+          # Merge distributed trace ids if present
+          unless env['HTTP_X_DDTRACE_PARENT_TRACE_ID'].nil? || env['HTTP_X_DDTRACE_PARENT_SPAN_ID'].nil?
```

Comment:
They are documented in http://www.rubydoc.info/github/DataDog/dd-trace-rb/ as the example for distributed tracing so I would say yes? I also want to add setting these in the `net/http` middleware which would mean you wouldn't even have to manually set them if it was included.